### PR TITLE
Log In/Out

### PR DESCRIPTION
**Pull request by binhpt-1239**
Log In And Log Out
- Add feature log in, log out
- add session, cookie
- add remember me

Screenshot from 2022-03-16 14-47-01
Screenshot from 2022-03-16 14-46-43
Screenshot from 2022-03-16 14-45-46

### CODE REVIEW

**Comment by binhpt-1239** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -16,22 +16,22 @@ jobs:
...
+#     environment:
+#       MYSQL_DATABASE: db_test
+#       MYSQL_USER: user_test
+#       MYSQL_PASSWORD: password_test
+#       MYSQL_ROOT_PASSWORD: password_test
+#   before_script:
+#   - bundle _2.1.4_ install --path vendor/bundle # _2.1.4_ if ruby 2.7.1 or 3.0.2
+#   script:
+#   - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+#   - bundle _2.1.4_ exec rspec # _2.1.4_ if ruby 2.7.1 or 3.0.2
```
Comment (nvatuan) [inline .sun-ci.yml:34]:
```suggestion
 - name: test:rspec
   stage: test
   image: manhbnt/ruby:2.7.1
   services:
   - image: mysql:5.7.22
     name: mysql_test
     environment:
       MYSQL_DATABASE: db_test
       MYSQL_USER: user_test
       MYSQL_PASSWORD: password_test
       MYSQL_ROOT_PASSWORD: password_test
   before_script:
   - bundle _2.1.4_ install --path vendor/bundle # _2.1.4_ if ruby 2.7.1 or 3.0.2
   script:
   - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
#   - bundle _2.1.4_ exec rspec # _2.1.4_ if ruby 2.7.1 or 3.0.2
```

**Comment by BaoKhanh99** [comment]
ok

File: .sun-ci.yml (hunk 1 above)
Comment (binhpt-1239) [reply .sun-ci.yml:34]: như vậy nó vẫn fails sun-ci á ông

Comment (vanvtt-0952) [reply .sun-ci.yml:34]: `bundle _2.1.4_ exec rspec` comment cái này thôi cậu

**Pull request merged**